clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4610,6 +4610,18 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
       CmdArgs.push_back(
           Args.MakeArgString(Twine("-fsycl-unique-prefix=") + UniqueID));
 
+    // Disable parallel for range-rounding for anything involving FPGA
+    auto SYCLTCRange = C.getOffloadToolChains<Action::OFK_SYCL>();
+    bool hasFPGA = false;
```

Comment:
As per the [naming convention](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly):
```suggestion
    bool HasFPGA = false;
```

Reply:
Thanks @AGindinson for caching this. Done